opentracing-tracerresolver/src/main/java/io/opentracing/contrib/tracerresolver/TracerResolver.java

```diff
@@ -95,10 +92,7 @@ public static Tracer resolveTracer() {
      * Reloads the lazily found {@linkplain TracerResolver resolvers} and the fallback resolver.
      */
     public static void reload() {
```

Comment:
Since `reload()` effectively became a no-op, should we deprecate it's use with an explanation why?